### DISCUSSION TITLE
!ALERT! OVERSEAS ACCOUNT DONATION SCAM!

### DISCUSSION THREAD

**Discussion by angry-mustard-it** [community/community#120386] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello guys, today i would like to show a scam of someone having to donate his accounts on OverSeas. This is an ongiong scam. I reported it to GitHub for more.

![Screenshot](https://github.com/community/community/assets/163648217/374400e6-3f60-420f-b7d5-13c12d3261db)

Viewing the page source "top233" will reveal it:  
_<title>Top出海-海外账号交易平台</title>
  
  "gmail邮箱购买,Github账号购买,vk账号,YouTuBe账号_

Translation:

_Top Going overseas-overseas account trading platform
Gmail email purchase, Github account purchase, vk account, YouTube account_

It is allegedly said, the same problem happened with Supercell's Clash of Clans, account donations, hackings, etc.
User @alalss gotten hacked by a scam known as  the Overseas account purchase, and said he wanted to purchase his social media accounts directly on overseas, for an approximate 200-3000 Yuan. Please! Report this!

He is also spamming on github gists. SHARING OR PURCHASING YOUR ACCOUNT BREAKS OUR TOS!

**Comment by ettaboyle** [comment, marked as answer]
Hi @angry-mustard-it, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by angry-mustard-it** [comment]
Thanks. I appreciate with that.
